src/C2_economic_functions.py

```diff
@@ -76,74 +76,148 @@ def crf(project_life, discount_factor):
     return crf
 
 
-def capex_from_investment(investment_t0, lifetime, project_life, discount_factor, tax):
+def capex_from_investment(
+    investment_t0, lifetime, project_life, discount_factor, tax, age_of_asset
+):
     """
-    Calculates the capital expenditures, also known as CapEx.
+     Calculates the capital expenditures, also known as CapEx.
+
+     CapEx represent the total funds used to acquire or upgrade an asset.
+     The specific capex is calculated by taking into account all future cash flows connected to the investment into one unit of the asset.
```

Comment:
What is "one unit of the asset" ? Could you provide and example? Or is it something trivial?